src/common/grammar.go

```diff
@@ -0,0 +1,62 @@
+package common
```

Comment:
`grammar.go`is an overloaded name, IMO. I opened this expecting some parsing constructs - [example grammar.go](https://github.com/golang/go/blob/master/src/mime/grammar.go)

Since this logic is only used in `alias.go`, it might make sense to move it all over there.